### PR TITLE
[Documentation][Book] When should you use Themes?

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no |
| Related tickets | mentioned in #7304 |
| License         | MIT |

Thanks to suggestions of @gabiudrescu! 👍

### CODE REVIEW

File: docs/book/themes/themes.rst [hunk 1]
```diff
@@ -4,7 +4,24 @@
 Themes
 ======
 
-Theming is a method of customizing how your channels look like in Sylius. Each channel can have a different tqheme.
+Theming is a method of customizing how your channels look like in Sylius. Each channel can have a different tcheme.
```
Comment (venyii) [inline docs/book/themes/themes.rst]: Almost, `tcheme` => `theme` 😄  🎉

Comment (TheMadeleine) [inline docs/book/themes/themes.rst]: Hahah xD Thx @venyii :D

**Comment by pjedrzejewski** [comment]
@venyii Thanks for the help with the review! We really need more eyes on the docs, @TheMadeleine is doing a great job, but this is a huge project... I appreciate your time!

**Comment by pjedrzejewski** [comment]
👍 👍 👍

**Pull request merged**